ros/src/twist_controller/dbw_node.py

```diff
@@ -100,5 +107,16 @@ def handleDBWEnabledMessage(self, dbw_enabled_message):
         rospy.loginfo(
             'Drive by Wire %s', "enabled" if self.dbw_enabled else "disabled")
 
+    def handle_target_velocity_message(self, twist_velocity_command_message):
+        twist = twist_velocity_command_message.twist
+        self.target_linear_velocity = twist.linear.x
+        self.target_angular_velocity = twist.angular.z
+
+    def handle_current_velocity_message(self, twist_current_velocity_message):
+        self.current_linear_velocity = msg.twist.linear.x
```

Comment:
Should this "msg" be "twist_current_velocity_message"?

Reply:
Ah, you're right - good catch! I wonder why that didn't show up as an error when I was running it. Ohh, I know why - I didn't have the simulator running, so I didn't get any active messages which would have highlighted this problem.

Will update and let you know for another look.

Reply:
Ready for re-review.